client/blindbox/tests/session.py

```diff
@@ -1,14 +1,14 @@
 from blindbox.requests  import SecureSession, requests
 
-with SecureSession("http://localhost:8080", True) as secure_session:
+with SecureSession("http://20.223.63.77:80", "./cce_policy.txt",True) as secure_session:
 
-    res = secure_session.post(endpoint="/enclave/predict", files={"audio": open("test.wav", "rb")})
+    res = secure_session.post(endpoint="/whisper/predict", files={"audio": open("test.wav", "rb")})
```

Comment:
The demo_server example does use the enclave endpoint, please either restore that or rename it in the demo server as well.

Reply:
Reverted this